apps/src/MusicController.js

```diff
@@ -71,9 +76,41 @@ var MusicController = function (audioPlayer, assetUrl, trackDefinitions) {
   /** @private {string} Name of track to play on load */
   this.playOnLoad_ = null;
 
+
+  /** @private {number} */
+  this.loopRandomWithDelay_ = loopRandomWithDelay;
+
+  /**
+   * @private {boolean} whether we stopped playing music due to video being
+   *          shown
```

Comment:
If you want to capture the double-meaning of this variable, could at least do it in this comment for now.